### DISCUSSION TITLE
Admin pages hang

### DISCUSSION THREAD

**Discussion by nicuznicuz** [globaleaks/globaleaks-whistleblowing-software#4106] [Q&A]
Hi everybody. While the remaining pages are working as expected, I cannot access the admin pages. After the authentication, the home page shows up, but any other click to the links is unresponsive; the page freezes probably waiting for the backend to answer, but nothing really happens. 

Logs are not really helpful, at least for me:

`2024-06-14 15:30:01+0200 [-] GlobaLeaks is now running and accessible at the following urls:
2024-06-14 15:30:01+0200 [-] - [HTTPS]: https://
2024-06-14 15:30:01+0200 [-] - [Tor]:  http://
2024-06-14 15:30:02+0200 [-] Starting factory _HTTP11ClientFactory(<function HTTPConnectionPool._newConnection.<locals>.quiescentCallback at 0x7fbe7fec2980>, <twisted.internet.endpoints._WrapperEndpoint object at 0x7fbe8000ce10>)
2024-06-14 15:30:02+0200 [-] Stopping factory _HTTP11ClientFactory(<function HTTPConnectionPool._newConnection.<locals>.quiescentCallback at 0x7fbe7fec2980>, <twisted.internet.endpoints._WrapperEndpoint object at 0x7fbe8000ce10>)`

Can you please give me a hint? 
Thank you in advance
Ciao
Nico

**Comment by evilaliv3** [comment]
Thank you for the question,

Actually the log line reported is not related.

Did you manage to identify the real cause? 

- Which version of GlobaLeaks are you using?
- How have you setup the software? 
- Do you have a link to the server that you can share?

**Comment by nicuznicuz** [comment]
Hi Giovanni. Thank you for answering.

Actually I realized that the issue happened only from a specific
client/browser (windows 10/Firefox 12x). I simply switched to another
client and the problem disappeared. After a while, and after a couple of
updates from the Globaleaks Debian repository, I did not have any other
issues, even with the original client. I guess that there was something on
the client side.

Ciao
Nico

Il giorno lun 8 lug 2024 alle ore 08:31 Giovanni Pellerano <
***@***.***> ha scritto:

> Thank you for the question,
>
> Actually the log line reported is not related.
>
> Did you manage to identify the real cause?
>
>    - Which version of GlobaLeaks are you using?
>    - How have you setup the software?
>    - Do you have a link to the server that you can share?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/globaleaks/discussions/4106#discussioncomment-9983757>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBP5X7YESMZ5SYDDCK3ILATZLIW5PAVCNFSM6AAAAABJKMT3RSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBTG42TO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Ciao
Nico
